api/v6/report_server.thrift

```diff
@@ -116,7 +116,8 @@ struct RunData {
   4: i64                       duration,     // Duration of the run (-1 if not finished).
   5: i64                       resultCount,  // Number of results in the run.
   6: string                    runCmd,       // The used check command.
-  7: map<DetectionStatus, i32> detectionStatusCount // Number of reports with a particular detection status.
+  7: map<DetectionStatus, i32> detectionStatusCount, // Number of reports with a particular detection status.
```

Comment:
This is an api change, so api version should be increased accordingly.

Reply:
I increased the version in these files:
- libcodechecker/version.py
- www/scripts/version.js

Do I have to change the api version in other files too?

Reply:
These two files should be enough.